budget-parse/src/app.ts

```diff
@@ -15,6 +15,31 @@ const checkIgnoreTransaction = (description: string) => {
   return ignoreTransactions.some((ignoreItem) => compareText.includes(ignoreItem.toLowerCase()));
 };
 
+const formatOriginalDescription = (input: string) => {
+  const capitalDescription =
+    input.charAt(0).toUpperCase() + input.slice(1).toLowerCase();
+
+  const newDescription = capitalDescription.replace(/\s\s+/g, ' ');
+  return newDescription
+}
+
+const createDescriptionString = (descriptions: string[]) => {
```

Comment:
This could probably be generalized to use in the debit section as well in the future. But length is not as much of a concern with that column